ansible_builder/main.py

```diff
@@ -81,11 +85,23 @@ def __init__(self, filename):
             with open(filename, 'r') as f:
                 y = yaml.load(f)
                 self.raw = y if y else {}
-        except FileNotFoundError:
-            sys.exit("""
-            Could not detect '{0}' file in this directory.
-            Use -f to specify a different location.
-            """.format(constants.default_file))
+        except (FileNotFoundError, yaml.parser.ParserError) as e:
+            if isinstance(e, yaml.parser.ParserError):
+                sys.exit(MessageColors.FAIL + textwrap.dedent("""
+                An error occured while parsing the definition file:
+                {0}
+                """).format(str(e)) + MessageColors.ENDC)
+            elif isinstance(e, FileNotFoundError):
+                sys.exit(MessageColors.FAIL + """
+                Could not detect '{0}' file in this directory.
+                Use -f to specify a different location.
+                """.format(constants.default_file) + MessageColors.ENDC)
```

Comment:
I think what you really want is

```python
except FileNotFoundError:
  ...
except yaml.parser.ParserError as e:
  ...
```

same thing functionally, just a matter of python standard practice